src/components/NewMessageForm/AdvancedInput/AdvancedInput.vue

```diff
@@ -73,6 +73,33 @@ import { searchPossibleMentions } from '../../../services/mentionsService'
 import Avatar from '@nextcloud/vue/dist/Components/Avatar'
 import Mention from '../../MessagesList/MessagesGroup/Message/MessagePart/Mention'
 
+/**
+ * Removes the default atwho style sheet added by the vue-at component.
+ *
+ * The rules in that style sheet are too broad and affect other elements
+ * besides those from the vue-at panel.
+ */
+function removeDefaultAtWhoStyleSheet() {
+	const styleElements = document.querySelectorAll('style')
+	for (let i = 0; i < styleElements.length; i++) {
+		const sheet = styleElements[i].sheet
+
+		if (sheet.cssRules.length !== 15) {
+			continue
+		}
+
+		for (const cssRule of sheet.cssRules) {
+			if (cssRule.cssText === '.atwho-view { color: rgb(0, 0, 0); border-radius: 3px; box-shadow: rgba(0, 0, 0, 0.1) 0px 0px 5px; min-width: 120px; z-index: 11110 !important; }') {
```

Comment:
this will not break at all with any update in the future :rofl: 

Maybe it would be better to check for `.atwho-view` only?